### DISCUSSION TITLE
Ist der USB-Anschluss am Smartgateway wirklich nur als Ladebuchse verdrahtet oder kann man ihn für andere Zwecke aktivieren?

### DISCUSSION THREAD

**Discussion by kuwagei** [Freetz-NG/freetz-ng#887] [Allgemein]
Habe mein Fritz!Smartgateway gefreetzt und möchte per chroot weitere Software ausführen. Da der interne Speicher sehr klein ist, kann ich das bisher nur, indem ich ein cifs-Share von der Fritz!Box verbinde und ein dort vorhandenes extfs-image loop-mounte. 
Weiß jemand, ob man den USB-Anschluss z.B. für einen Speicherstick aktivieren kann?
Ein USB-Treiber ist ja vorhanden und wird offenbar intern für Zigbee verwendet.

**Comment by fda77** [comment]
Schraub das Ding mal auf und schau welche Hardware für USB drin ist. Oder schau was bei dmesg/syslog für Treiber geladen werden. Falls welche geladen werden, nur zum aufladen könnte aus auch ohne gehen. Oder ldmod/lsusb in /proc/ usw
Dann nach dem Bauteil googeln und schauen was es kann
